src/components/Templates/GetNfscCard/GetNfscCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Chip } from '../../Core/Chip'
+import { cn } from '../../../utils/cn';
+
+export type GetNfscCardProps = React.ComponentProps<'div'> &
+  Partial<{
+    onClickChip: any,
+    value: number
+  }>;
+
+let cardTitle = "Get $NFSC";
+
+export function GetNfscCard(
+  { onClickChip, value, children, className, ...props }: GetNfscCardProps) {
+    let chips = [20, 50, 100].map(
+      (chipValue) => (<Chip selected={value === chipValue} onSelect={() => onClickChip(chipValue)} >{chipValue}</Chip>));
+  return (
+    <div className={cn('flex flex-col justify-start items-center rounded-3.75 gap-3 py-9 px-11.5 bg-black-1000 w-[613px] h-[581px]', className)} {...props}>
```

Comment:
we shouldn't use strict sizing on parts of the layout, strict sizing could be used on small components or icons.
in most cases you might need strict sizing in height only, but depending on the case it's best to use max and min width.
so for this example, I think the design intention was `max-w-[613px] min-h-[581px]`

Reply:
Now using max-w and min-h instead of w- and h-